frontend/src/components/EditView.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import MonacoEditor from './MonacoEditor'
 import { useSelector } from 'react-redux'
-import { RootState } from '../store/store'
+import { RootState, RepoFile } from '../types'
 
-interface RepoFile {
-    filename: string,
-    content: string,
+interface Props {
```

Comment:
Could this be made more generic and usable in many different components?

Reply:
I am referring to the declaration of the Props-interface outside of the types-directory

Reply:
The way I see it is that the point of having these props interfaces is to define specifically what fields props need to have, so trying to make them generic right away would be pointless. Of course if multiple components end up needing similar props we can make some generic interfaces.